set-versions.bat

```diff
@@ -4,14 +4,14 @@ REM Set these version variables and run the script to change all the files which
 REM The site extension version
 set major=0
 set minor=3
-set patch=8
+set patch=9
 set version_postfix=-prerelease
 
 REM The agent version
 set agent_version=7.18.1
```

Comment:
Agent it up to 7.22, should we update?

Reply:
Yeah good idea, updated in 4694f5f61e3e30d75002fde2101deba037f59953 . This will reset my testing which should not take too long